status/status.go

```diff
@@ -120,14 +120,14 @@ func FromProto(s *spb.Status) *Status {
 }
 
 // FromError returns a Status representing err if it was produced from this
-// package or has a method Status() *Status. Otherwise, ok is false and a
+// package or has a method `GRPCStatus() *Status`. Otherwise, ok is false and a
```

Comment:
Do these backticks translate correctly to monospace in GoDoc?

Reply:
No, they will look like literal backticks. I would be fine with single or double quotes here as well, but I would prefer something that visually groups the method name with its return value for clarity.